<ui version="4.0" >
 <class>cs8WagoConfigMainWindow</class>
 <widget class="QMainWindow" name="cs8WagoConfigMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Swiss System Plastic Config Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="tabPosition" >
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>System Node</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="cs8WagoNodeWidget" native="1" name="SystemNode" />
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btSystemRefresh" >
            <property name="text" >
             <string>&amp;Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btSystemProposal" >
            <property name="text" >
             <string>&amp;Propose IO Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btSystemSave" >
            <property name="text" >
             <string>&amp;Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ta" >
       <attribute name="title" >
        <string>User Node</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="cs8WagoNodeWidget" native="1" name="UserNode" />
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btUserRefresh" >
            <property name="text" >
             <string>&amp;Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btUserProposal" >
            <property name="text" >
             <string>&amp;Propose IO Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btUserSave" >
            <property name="text" >
             <string>&amp;Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Modbus File</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="cs8ModbusWidget" native="1" name="modbusView" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open" />
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionRecommission_Wago_Coupler" />
    <addaction name="separator" />
    <addaction name="action_Backup_Configuration" />
    <addaction name="action_Restore_Configuration" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
    <addaction name="action_About_Qt" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menuTools" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Open" >
   <property name="text" >
    <string>&amp;Upload Modbus File from Controller</string>
   </property>
  </action>
  <action name="actionRecommission_Wago_Coupler" >
   <property name="text" >
    <string>Recommission Wago Coupler</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_About_Qt" >
   <property name="text" >
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="action_Backup_Configuration" >
   <property name="text" >
    <string>&amp;Backup Configuration</string>
   </property>
  </action>
  <action name="action_Restore_Configuration" >
   <property name="text" >
    <string>&amp;Restore Configuration</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cs8ModbusWidget</class>
   <extends>QWidget</extends>
   <header>cs8ModbusComponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cs8WagoNodeWidget</class>
   <extends>QWidget</extends>
   <header>cs8ModbusComponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>cs8WagoConfigMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
